lib/src/merged_tree.rs

```diff
@@ -952,37 +952,36 @@ impl<'matcher> TreeDiffStreamImpl<'matcher> {
     }
 
     /// Gets the given tree if `value` is a tree, otherwise an empty tree.
```

Comment:
nit: Maybe "tree" -> "trees" now, and "value" should have been "values" from before

Reply:
Inserted new commit to fix the doc comments. There are two instances.